R/GRN.R

```diff
@@ -15,7 +15,8 @@
 #' data('DEGsmatrix')
 #' data('dataFilt')
 #' dataDEGs <- DEGsmatrix
-#' dataGRN <- GRN(TFs = rownames(dataDEGs)[1:10],
+#' set.seed(317)
```

Comment:
I added the seed here so we know what seed generated the example data and so that we would know how to replicate the example data. Should I place it somewhere else?